editor/filesystem_dock.cpp

```diff
@@ -3581,6 +3581,11 @@ void FileSystemDock::_tree_empty_click(const Vector2 &p_pos, MouseButton p_butto
 
 void FileSystemDock::_tree_empty_selected() {
 	tree->deselect_all();
+	current_path = "";
+	current_path_line_edit->set_text(current_path);
```

Comment:
This feels like it should be associated to its own method that sets both at once, but I don't know about the internals of this

Reply:
I counted 2 other places that could use the method. In other cases the path and text are assigned separately, 
https://github.com/godotengine/godot/blob/46c495ca21f40f57a7fb9c7cde6143738f1652d4/editor/filesystem_dock.cpp#L1742-L1748
so the method would be not very useful.